meterfs/test_meterfs_common.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs tests common part
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ *
+ * %LICENSE%
+ */
+
+/* TODO: Decide what should be in test_common struct */
+#ifndef TEST_METERFS_COMMON_H
+#define TEST_METERFS_COMMON_H
+
+#include <unistd.h>
+#include <string.h>
+
+#include "platform/test_file.h"
+#include "unity_fixture.h"
+
+struct {
+	size_t flashsz;
+	size_t sectorsz;
+	size_t sectors;
+	size_t filesz;
+	size_t recordsz;
+	size_t recordcnt;
+	int ids[255];
+	char buff[21];
+} test_common;
```

Comment:
In case of test it's maybe not that bad but I think that declaring variables in .h file is like an asking for a troubles. 

If you accidentially include this in two file which will be compiled into one binary weird things just happened.
GCC 10 enables -fno-common by default causing link error in that case.
GCC <= 9 will (probably silently) ignore one of the variable and code will be point to one of them.

If you would like to continue I will create some test_ctx_t and pass it to function or if don't want, declare it as extern and define in .c file.

Also – I don't like it for the reason that probably some of test won't need that ids[255]. And it will increase binary size.

I know it's test only.

Reply:
Done. Every test group has their own common struct and I added `file_info_t` typedef.